python/sdk/merlin/model.py

```diff
@@ -333,12 +334,33 @@ def list_version(self) -> List['ModelVersion']:
 
         :return: list of ModelVersion
         """
+        result: List['ModelVersion'] = []
+        versions, cursor = self.list_version_pagination()
+        result = result + versions
+        while cursor != "":
+            versions, cursor = self.list_version_pagination(cursor=cursor)
+            result = result + versions
+            sleep(0.05)
+        return result
+
+    def list_version_pagination(self, limit=DEFAULT_MODEL_VERSION_LIMIT, cursor="", search="") -> Tuple[List['ModelVersion'], str]:
```

Comment:
In what case this function will be useful for?

Reply:
it is called on `list_version` function, to list all the versions

Reply:
In that case it will be better to make it private